topics/go/algorithms/strings/permuation/permutation.go

```diff
@@ -0,0 +1,27 @@
+package strings
+
+import (
+	"sort"
+	"strings"
+)
+
+// IsPermutation check if two
+// strings are permutations.
+func IsPermutation(str1, str2 string) bool {
+
+	// If the length are not equal they cannot be permutation.
+	if len(str1) != len(str2) {
+		return false
+	}
+
+	// Create a slice for each input string that
+	//is split on every character of the string.
+	s1 := strings.Split(str1, "")
```

Comment:
Look at the comment on line 12, that is a good length before the break.

// <space> is split